adam/curriculum/verbs_with_dynamic_prepositions_curriculum.py

```diff
@@ -474,6 +483,216 @@ def _go_over_under_path_template(
     )
 
 
+def _x_roll_beside_y_template(
```

Comment:
@chris-jenkins-isi 
The intent behind separating them out is so that future curriculum that only wants "roll beisde" in their curriculum are easier to generate because you don't have to duplicate the template code. They can just import this template creator and pass in the TemplateObjects.